nibabel/minc1.py

```diff
@@ -307,6 +311,26 @@ def from_file_map(klass, file_map):
             data = klass.ImageArrayProxy(minc_file)
         return klass(data, affine, header, extra=None, file_map=file_map)
 
+    @classmethod
+    def is_image(klass, filename, sniff=None):
+        ftypes = dict(klass.files_types)
+        froot, ext, trailing = splitext_addext(filename, klass._compressed_exts)
+        lext = ext.lower()
+
+        if lext not in ftypes.values():
+            return False, sniff
+
+        fname = froot + ftypes['header'] if 'header' in ftypes else filename
+        if not sniff:
+            with BinOpener(fname, 'rb') as fobj:
+                sniff = fobj.read(4)
+
+        return klass._minctest(sniff), sniff
```

Comment:
The reason this was an issue is because minc1 and minc2 both use the `MincHeader`, which then needs to give a different response based on whether a `Minc1Image` or `Minc2Image` called it.

There's probably a better way to do it.